addons/project/models/project_project.py

```diff
@@ -51,24 +51,17 @@ def _search_is_favorite(self, operator, value):
             raise NotImplementedError(_('Operation not supported'))
         return [('favorite_user_ids', 'in' if (operator == '=') == value else 'not in', self.env.uid)]
 
+    @api.depends('favorite_user_ids')
```

Comment:
I don't think adding the depends is necessary, there is no way `is_favorite` would have to be recomputed